### ISSUE TITLE
Puma integration plugin installation issue: ExpiredMetadataError

### ISSUE THREAD

**Issue by gopivalleru** [DataDog/integrations-extras#2301]
Getting `'dd-integrations-core-wheels-build-stable.datadoghq.com': ExpiredMetadataError("Metadata 'root' expired on Tue Oct 17 17:47:17 2023 (UTC).")` while trying to install puma integration.

**Additional environment details (Operating System, Cloud provider, etc):**

OS: Ubuntu 18.04.5 LTS
agent version: Agent 7.39.2 - Commit: f6f7bc9 - Serialization version: v5.0.26 - Go version: go1.17.11

**Steps to reproduce the issue:**
1. Run the integration install command which should reproduce the issue.

```bash

sudo /opt/datadog-agent/bin/agent/agent integration install --third-party datadog-puma==1.2.0 --allow-root
Traceback (most recent call last):
  File "/opt/datadog-agent/embedded/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/datadog-agent/embedded/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/downloader/__main__.py", line 9, in <module>
    sys.exit(download())
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/downloader/cli.py", line 118, in download
    tuf_downloader = TUFDownloader(
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/downloader/download.py", line 106, in __init__
    self.__updater.refresh()
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/tuf/client/updater.py", line 1080, in refresh
    self._update_root_metadata(root_metadata)
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/tuf/client/updater.py", line 1147, in _update_root_metadata
    self._update_metadata('root', DEFAULT_ROOT_UPPERLENGTH,
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/tuf/client/updater.py", line 1696, in _update_metadata
    self._get_metadata_file(metadata_role, remote_filename,
  File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/tuf/client/updater.py", line 1621, in _get_metadata_file
    raise exceptions.NoWorkingMirrorError(file_mirror_errors)
tuf.exceptions.NoWorkingMirrorError: No working mirror was found:
  'dd-integrations-core-wheels-build-stable.datadoghq.com': ExpiredMetadataError("Metadata 'root' expired on Tue Oct 17 17:47:17 2023 (UTC).")
Error: error when downloading the wheel for datadog-puma 1.2.0: error running command: exit status 1
```

**Describe the results you received:**
I'm getting 'dd-integrations-core-wheels-build-stable.datadoghq.com': ExpiredMetadataError("Metadata 'root' expired on Tue Oct 17 17:47:17 2023 (UTC).")

**Describe the results you expected:**
It is supposed to install puma integration.

**Additional information you deem important (e.g. issue happens only occasionally):**
The issue is always happening.

**Comment by FlorentClarret** [comment]
Hello, yes these metadata expired and this version of the Agent is not able to check the new ones. We fixed this issue with Agent 7.43.0

**Comment by gopivalleru** [comment]
Thanks. We'll upgrade to latest version.
